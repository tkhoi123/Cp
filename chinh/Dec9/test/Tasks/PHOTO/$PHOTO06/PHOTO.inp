67
66 60 21 64 56 52 51 8 50 20 61 47 29 9 35 62 6 46 3 1 48 30 2 39 5 38 22 44 19 43 42 65 18 45 28 53 55 36 14 63 23 59 26 4 41 25 11 24 7 33 40 10 54 31 32 67 37 58 13 57 27 17 34 15 12 16 49 
66 54 60 64 10 56 46 51 8 20 61 1 29 9 58 25 7 35 62 6 48 3 30 2 39 23 5 38 19 43 44 47 42 21 65 32 18 53 36 40 22 50 14 55 63 59 26 28 41 24 33 31 4 67 37 27 45 34 52 13 57 17 15 12 16 11 49 
54 60 64 10 56 52 46 51 50 61 9 24 7 35 62 6 3 63 1 48 30 2 39 23 5 38 22 44 8 19 43 36 47 21 65 18 53 14 20 55 26 28 25 11 33 40 29 31 32 66 4 67 45 34 58 57 27 17 42 37 15 59 12 41 16 49 13 
66 54 10 56 52 46 8 38 5 6 50 20 61 29 9 7 35 62 3 1 16 48 64 30 39 23 22 44 43 47 42 21 65 19 18 36 14 55 63 59 26 28 41 25 11 24 33 40 57 31 32 4 37 2 45 53 34 58 51 13 27 17 15 60 12 49 67 
66 54 60 64 10 52 9 17 18 46 51 8 50 30 20 29 7 26 6 3 1 2 48 39 23 56 5 38 22 61 44 19 35 43 47 42 21 53 36 14 55 65 63 59 28 41 25 11 24 33 40 31 32 4 67 37 45 34 58 62 13 57 27 15 12 16 49 
