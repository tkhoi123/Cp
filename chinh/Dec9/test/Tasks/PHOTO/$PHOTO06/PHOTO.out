66 54 60 64 10 56 52 46 51 8 50 20 61 29 9 7 35 62 6 3 1 48 30 2 39 23 5 38 22 44 19 43 47 42 21 65 18 53 36 14 55 63 59 26 28 41 25 11 24 33 40 31 32 4 67 37 45 34 58 13 57 27 17 15 12 16 49 